src/gui/lineedit.cpp

```diff
@@ -29,20 +29,41 @@
 
 #include "lineedit.h"
 
+#include <chrono>
+
 #include <QAction>
 #include <QKeyEvent>
+#include <QTimer>
 
 #include "base/global.h"
 #include "uithememanager.h"
 
+using namespace std::chrono_literals;
+
+namespace
+{
+    const std::chrono::milliseconds FILTER_INPUT_DELAY {750};
```

Comment:
To me that "feels" too slow. If this was my first time searching, i would think "oops i didn't press enter" or "wow searching is horribly slow in this program".

Reply:
Are you able to suggest a value? Keep in mind that the users whom reported the issues had to wait several seconds for an update. I mean it would be costly for them if they got it wrong and don't have enough time to correct it.

Reply:
We obviously can't cover all the cases.
I would assume that it should not start searching while the user enters the desired value. I would start with 200 milliseconds. In the future, you may even have to make it customizable.

Reply:
The webui was using 400ms for a long time so now I changed all UI to use the same 400ms.
Hope it is better now.